src/Components/Aspire.Pomelo.EntityFrameworkCore.MySql/AspireEFMySqlExtensions.cs

```diff
@@ -63,6 +67,22 @@ public static partial class AspireEFMySqlExtensions
 
         builder.Services.AddDbContextPool<TContext>(ConfigureDbContext);
 
+        const string resilienceKey = "Microsoft.Extensions.Hosting.AspireEFMySqlExtensions.ServerVersion";
+        builder.Services.AddResiliencePipeline(resilienceKey, static builder =>
```

Comment:
Can we follow the same pattern as #2400 here?

Reply:
Yes, will do.

Reply:
I did in https://github.com/mysql-net/aspire/pull/2. If you want to take that.

Reply:
Thanks! Was tied up in meetings all morning so haven't been able to get to this. Appreciate it.